src/pages-and-resources/app-settings-modal/AppSettingsModal.jsx

```diff
@@ -124,13 +122,18 @@ function AppSettingsModal({
             onSubmit={handleFormSubmit}
           >
             {(formikProps) => (
-              <Form onSubmit={formikProps.handleSubmit}>
+              <Form
+                onSubmit={formikProps.handleSubmit}
+                style={{
+                  width: '100%', height: '100%', display: 'flex', 'flex-flow': 'column',
```

Comment:
this can be replaced with => `w-100 h-100 d-flex flex-column`

Reply:
fixed it